dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -242,9 +241,6 @@ public Workflow restub(@ApiParam(hidden = true) @Parameter(hidden = true, name =
         if (workflow.isIsChecker()) {
             throw new CustomWebApplicationException("A checker workflow cannot be restubed.", HttpStatus.SC_BAD_REQUEST);
         }
-        if (!workflow.getConceptDois().isEmpty()) {
```

Comment:
Hesitant about relaxing this in general in case someone discovers this and starts depending on it. 
Can we limit to curators and admins or in some other way that is not available to regular users?

Reply:
Per our short Slack discussion, I've changed the PR so that workflows with DOIs can be restubbed on non-prod platforms (but not on prod).